samples/csharp/MicrosoftEduGraphSamples/workflows/Assignment.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+
+namespace MicrosoftEduGraphSamples.Workflows
+{
+    /// <summary>
+    /// Contains all the workflows related to Assignment
+    /// </summary>
+    internal class Assignment
+    {
+
+        private readonly IConfiguration _config;
+
+        public Assignment(IConfiguration configuration)
+        {
+            this._config = configuration;
+        }
+        /// <summary>
+        /// Workflow to get assignments from all the classes which are not archived
+        /// </summary>
+        public void AssignmentsFromNotArchivedClasses ()
+        {
+            try
+            {
+                List<EducationAssignment> assignmentsFromNonArchivedClasses = new List<EducationAssignment>();
+
+                // Get a Graph client using delegated permissions
+                var graphClient = MicrosoftGraphSDK.GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["teacherAccount"], _config["password"]);
+
+                //Call to get user classes
+                var JoinedTeams = MicrosoftGraphSDK.Team.GetJoinedTeams(graphClient);
+
+                //Check to iterate over all classes
+                foreach(var team in JoinedTeams.Result) 
+                {
+                    // Verify if isArchived property = false
+                    if (team.IsArchived==false) 
```

Comment:
please change the condition to

if (!team.IsArchived)

Reply:
Thanks for your comment, this change cannot be included since !team is not working with boolean data type